cmd/create/cluster/cmd.go

```diff
@@ -2923,6 +3066,27 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 		command += fmt.Sprintf(" --audit-log-arn %s", *spec.AuditLogRoleARN)
 	}
 
+	if !reflect.DeepEqual(spec.DefaultIngress, ocm.DefaultIngressSpec{}) {
+		if len(spec.DefaultIngress.RouteSelectors) != 0 {
+			command += fmt.Sprintf(" --%s \"", defaultIngressRouteSelectorFlag)
+			for k, v := range spec.DefaultIngress.RouteSelectors {
+				command += fmt.Sprintf("%s=%s", k, v)
+			}
+			command += "\""
```

Comment:
If there is more than one key value pair, a comma separated list is not properly made here.

Reply:
Example (account id omitted)
```
? Route Selector for ingress (optional): foo=bar,test=case                                                                                         
? Excluded namespaces for ingress (optional): alpha,beta,gamma,delta                                                                               
? Wildcard Policy (optional, choose 'none' to skip selection): WildcardsDisallowed                                                                 
? Namespace Ownership Policy (optional, choose 'none' to skip selection): Strict                                                                   
I: Creating cluster 'rc-test'                                                                                                                      
I: To create this cluster again in the future, you can run:                                                                                        
   rosa create cluster --cluster-name rc-test --sts --role-arn arn:aws:iam::~:role/ManagedOpenShift-Installer-Role --support-role-arn ar
n:aws:iam::~:role/ManagedOpenShift-Support-Role --controlplane-iam-role arn:aws:iam::~:role/ManagedOpenShift-ControlPlane-Rol
e --worker-iam-role arn:aws:iam::~/ManagedOpenShift-Worker-Role --operator-roles-prefix rc-test-k9a2 --region us-east-1 --version 4
.12.22 --replicas 2 --compute-machine-type m5.xlarge --machine-cidr 10.0.0.0/16 --service-cidr 172.30.0.0/16 --pod-cidr 10.128.0.0/14 --host-prefix
 23 --default-ingress-route-selector "foo=bartest=case" --default-ingress-excluded-namespaces alpha,beta,gamma,delta --default-ingress-wildcard-pol
icy WildcardsDisallowed --default-ingress-namespace-ownership-policy Strict
```

Reply:
Oh, missed the comma lol. Thanks!